apps/cookbook/src/app/examples/menu-example/examples/selectable.ts

```diff
@@ -1,12 +1,15 @@
 import { Component } from '@angular/core';
+import { ToastConfig, ToastController } from '@kirbydesign/designsystem/toast';
 
 const config = {
   selector: 'cookbook-menu-selectable-example',
-  template: `<kirby-menu [closeOnSelect]="true">
-  <kirby-item [selectable]="true">
+  template: `<kirby-menu>
```

Comment:
With closeOnSelect inverted, the advanced items example should have closeOnSelect set to false to make sense 
 
<img width="310" alt="image" src="https://user-images.githubusercontent.com/42470636/234563078-0d8a1482-2879-43c3-9d6f-99c3055c9f60.png">

Reply:
Good catch! Fixed:

![image](https://user-images.githubusercontent.com/9556693/234575149-89227b44-4881-4521-8ec1-afbcf357acc5.png)